it/controllers/propertyLinking/UploadControllerISpec.scala

```diff
@@ -0,0 +1,1417 @@
+      commonEvidencePageTests(language = Welsh,
+        evidenceType = "WATER_RATE",
+        userIsAgent = false,
+        relationship = "Occupier",
+        backLink = backToOtherEvidenceHref,
+        fromCya = false)
+    }
+
+    "display the correct content in Welsh for an IP that is uploading a utility bill" which {
+      commonEvidencePageTests(language = Welsh,
+        evidenceType = "UTILITY_RATE",
+        userIsAgent = false,
+        relationship = "Owner",
+        backLink = backToOtherEvidenceHref,
+        fromCya = false)
+    }
+
+    "display the correct content in Welsh for an IP that is uploading other evidence (Occupier)" which {
+      otherEvidencePageTests(language = Welsh, relationship = "Occupier", userIsAgent = false, fromCya = false)
+    }
+
+    "display the correct content in Welsh for an IP that is uploading other evidence (Owner)" which {
+      otherEvidencePageTests(language = Welsh, relationship = "Owner", userIsAgent = false, fromCya = false)
+    }
+
+    "display the correct content in Welsh for an IP that is uploading other evidence (Owner and occupier)" which {
+      otherEvidencePageTests(language = Welsh, relationship = "Owner and occupier", userIsAgent = false, fromCya = false)
+    }
+
+    "display the correct content in Welsh for an IP that has uploaded a Lease (from CYA)" which {
+      commonEvidencePageTests(language = Welsh,
+        evidenceType = "LEASE",
+        userIsAgent = false,
+        relationship = "Occupier",
+        backLink = backToEvidenceQuestionHref,
+        fromCya = true)
+    }
+
+    "display the correct content in Welsh for an IP that has uploaded a business rates bill (from CYA)" which {
+      commonEvidencePageTests(language = Welsh,
+        evidenceType = "RATES_BILL",
+        userIsAgent = false,
+        relationship = "Owner",
+        backLink = backToEvidenceQuestionHref,
+        fromCya = true)
+    }
+
+    "display the correct content in Welsh for an IP that uploaded a lease and accesses the 'Other evidence page' (from CYA)" which {
+      otherEvidencePageTests(language = Welsh, relationship = "Owner and occupier", userIsAgent = false, fromCya = true)
+    }
+
+    // Agent - Welsh
+    "display the correct content in Welsh for an Agent that is uploading a business rates bill" which {
+      commonEvidencePageTests(language = Welsh,
+        evidenceType = "RATES_BILL",
+        userIsAgent = true,
+        relationship = "Owner",
+        backLink = backToEvidenceQuestionHref,
+        fromCya = false)
+    }
+
+    "display the correct content in Welsh for an Agent that is uploading a lease" which {
+      commonEvidencePageTests(language = Welsh,
+        evidenceType = "LEASE",
+        userIsAgent = true,
+        relationship = "Occupier",
+        backLink = backToEvidenceQuestionHref,
+        fromCya = false)
+    }
+
+    "display the correct content in Welsh for an Agent that is uploading a licence to occupy" which {
```

Comment:
this is just a test heading not a test (i think you need to delete the line below)

Reply:
Good spot, I will fix that